components/list-group.scss

```diff
@@ -0,0 +1 @@
+@import "../scss/components/list-group";
```

Comment:
This file needs fonts, then I think we're good to go!

Reply:
done